docs/concepts/python-versions.md

```diff
@@ -280,10 +280,9 @@ during `uv python install`.
 By default, uv will attempt to use Python versions found on the system and only download managed
 interpreters when necessary.
 
-The [`python-preference`](../reference/settings.md#python-preference) option can be used to adjust
-this behavior. By default, it is set to `managed` which prefers managed Python installations over
-system Python installations. However, system Python installations are still preferred over
-downloading a managed Python version.
+The `python-preference` option can be used to adjust this behavior. By default, it is set to
```

Comment:
We didn't remove this option from the settings reference too, did you mean to remove this link?